vhdbuilder/packer/components.json

```diff
@@ -306,7 +292,7 @@
         "v1.8.0.2"
       ],
       "multiArchVersions": [
-        "v1.10.0"
```

Comment:
could you verify this change? seems https://github.com/Azure/AgentBaker/pull/1395/files did not touch it

Reply:
yes, you're right. azuredisk-csi:v1.9.0 is the minimum version with 'multi-arch'. I will use 1.9.0 instead of 1.10.0.